tests/e2e/testsLibrary/WorksapceHandlingTests.ts

```diff
@@ -0,0 +1,34 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { NewWorkspace, CLASSES, Dashboard } from '..';
+import { e2eContainer } from '../inversify.config';
+
+const dashboard: Dashboard = e2eContainer.get(CLASSES.Dashboard);
+const newWorkspace: NewWorkspace = e2eContainer.get(CLASSES.NewWorkspace);
+
+export function createAndOpenWorkspace(workspaceName : string, stack: string) {
+    test('Open \'New Workspace\' page', async () => {
```

Comment:
Minor improvement - you can use backticks (\`). This way you can use normal and double quotes inside the string as well as variables.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals